### PR TITLE
add sentry error logging

### PR DESCRIPTION
**Pull request by adrianjost**
# Issue: [SC-2032](https://ticketsystem.schul-cloud.org/browse/SC-2032)

## Description

- Add Sentry Error Reporting

### CODE REVIEW

File: deploy/compose-client.dummy [hunk 1]
```diff
@@ -18,6 +17,7 @@ services:
       - TZ=Europe/Berlin
       - LEGACY_CLIENT_URL=http://client:3100
       - API_URL=https://api.test.schul-cloud.org
+      - SENTRY_DSN=https://18a7cbf1a7ef427ebcfc38429d18ba40@sentry.schul-cloud.dev/7
```
Comment (tofixx) [inline deploy/compose-client.dummy]: would not use that locally, please update the token asap and remove it here :)

File: src/plugins/sentry.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Theme from "@theme/config";
+
+export default ({ app, store }) => {
+	const { user } = store.state.auth;
+	app.$sentry.configureScope((scope) => {
+		scope.setLevel("warn");
+		scope.setTag("frontend", true);
+		scope.setTag("theme", Theme.name);
+		scope.setTag("domain", window.location.host);
```
Comment (tofixx) [inline src/plugins/sentry.js]: use SC_DOMAIN env instead?
is there no server part?
set git sha and release version

File: deploy/compose-client.dummy (hunk 1 above)
Comment (adrianjost) [reply deploy/compose-client.dummy]: thats not locally. That is our test system docker-compose file.

File: src/plugins/sentry.js (hunk 2 above)
Comment (adrianjost) [reply src/plugins/sentry.js]: This is only the client part where we have no server

**Pull request merged**